src/components/molecules/BenefitFilter.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ * This component displays three navigation buttons which filters out the result of benefits
+ */
+export function BenefitFilter(props) {
+  return (
+    <div>
+      <p className="text-gray-700 leading-none font-bold">{props.text}</p>
+      <div className="flex w-9/12">
+        <ActionButton
+          type="filter"
+          text={props.eligible}
+          count={props.eligibleCount}
+          invert={true}
+        />
+        <ActionButton
+          type="filter"
+          text={props.help}
+          count={props.helpCount}
+          invert={true}
+        />
+        <ActionButton
```

Comment:
Where are the onClick events for these buttons? Need to pass onClicks to be able to pass callbacks which will alter redux or state, otherwise how do these components work.

Reply:
One more thing before I forget, could you please add to the story the different states of when these three buttons are selected ?